tests/tables-test.sps

```diff
@@ -450,7 +450,7 @@
                   '(169 144 121 0 1 4 9 16 25 36 49 64 75 81)))
       '(13 12 11 0 1 2 3 4 5 -1 -1 8 -1 -1))
 
-(let ((ht-eg (hash-table number-comparator 1 1 4 2 9 3 16 4 25 5 64 8)))
+#;(let ((ht-eg (hash-table number-comparator 1 1 4 2 9 3 16 4 25 5 64 8)))
```

Comment:
The hash table is created with `hash-table` using the default #f value for mutable, hence it is not mutable.